src/ast/ast_build_filter_tree.h

```diff
@@ -10,4 +10,6 @@
 #include "../graph/query_graph.h"
 #include "../filter_tree/filter_tree.h"
 
-FT_FilterNode *AST_BuildFilterTree(AST *ast);
\ No newline at end of file
+void AST_ConvertFilters(FT_FilterNode **root, const cypher_astnode_t *entity);
```

Comment:
Is this a new function?

Reply:
Not new, but it was formerly not exposed.